Makefile

```diff
@@ -957,6 +970,109 @@ topo/us-%-cities.json: geojson/%/cities.geojson
+	mkdir -p $(dir $@)
+	cat data/state-centroids.geojson \
+		| ./reproject-geojson \
+		| ./normalize-properties postal:statePostal name:name \
+		> $@
+
+geojson/albers/%.geojson: geojson/%.geojson
+	mkdir -p $(dir $@)
+	cat $^ \
+		| ./reproject-geojson \
+		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
+		> $@
+
+geojson/albers/state-bounds.json: geojson/albers/states.geojson
+	cat $^ | ./extract-projected-bounds > $@
+
+election-results/2012.csv:
+	mkdir -p $(dir $@)
+	data/ap-to-csv data/2012-county-results.json data/2012-district-results.json > $@
+
+election-results/2012-state-centroids.geojson: geojson/albers/state-centroids.geojson
+	cat geojson/albers/state-centroids.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=statePostal --geojsonField=statePostal election-results/2012.csv \
+		| ./flatten-geojson \
+		> $@
+
+election-results/2012-%.geojson: geojson/albers/%.geojson
+	cat geojson/albers/$*.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=id --geojsonField=id election-results/2012.csv \
+		| ./add-density-property voteCount \
+		| ./flatten-geojson \
+		> $@
+
+election-results/2012-%-10m.geojson: geojson/albers/us-10m/%.geojson
+	cat geojson/albers/us-10m/$*.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=id --geojsonField=id election-results/2012.csv \
+		| ./add-density-property voteCount \
+		| ./flatten-geojson \
+		> $@
+
+tiles/%-results.z0-3.mbtiles: election-results/%.csv \
+	election-results/%-states-10m.geojson \
+	election-results/%-counties-10m.geojson \
+	election-results/%-districts-10m.geojson \
+	election-results/%-state-centroids.geojson
+	mkdir -p $(dir $@)
+	tippecanoe --projection EPSG:3857 \
+		--named-layer=states:election-results/$*-states-10m.geojson \
+		--named-layer=counties:election-results/$*-counties-10m.geojson \
+		--named-layer=districts:election-results/$*-districts-10m.geojson \
+		--named-layer=state-centroids:election-results/$*-state-centroids.geojson \
+		--read-parallel \
+		--no-polygon-splitting \
+		--maximum-zoom=3 \
+		--drop-rate=0 \
+		--name=$*-results \
+		--output $@
+
+tiles/%-results.z4-10.mbtiles: election-results/%.csv \
+	election-results/%-states.geojson \
+	election-results/%-counties.geojson \
+	election-results/%-districts.geojson \
+	election-results/%-state-centroids.geojson
+	mkdir -p $(dir $@)
+	tippecanoe --projection EPSG:3857 \
+		--named-layer=states:election-results/$*-states.geojson \
+		--named-layer=counties:election-results/$*-counties.geojson \
+		--named-layer=districts:election-results/$*-districts.geojson \
+		--named-layer=state-centroids:election-results/$*-state-centroids.geojson \
+		--read-parallel \
+		--no-polygon-splitting \
+		--minimum-zoom=4 \
+		--maximum-zoom=10 \
+		--drop-rate=0 \
+		--name=$*-results \
+		--output $@
+
+tiles/%-results.mbtiles: tiles/%-results.z0-3.mbtiles tiles/%-results.z4-10.mbtiles
+	tile-join -f -o $@ $^
```

Comment:
^ these three targets now make two different tilesets for the different zoom levels, and then merge them together.  tile-join doesn't support inputs w/ overlapping tiles, so important that the tilesets we're merging are disjoint